dicoogle/src/main/java/pt/ua/dicoogle/server/web/servlets/search/DumpServlet.java

```diff
@@ -58,11 +55,13 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
         if (StringUtils.isEmpty(uid)) {
             resp.sendError(400, "No uid supplied");
         }
-        
+
         String[] providerArr = req.getParameterValues("provider");
-        List<String> providers = (providerArr == null)
-                ? PluginController.getInstance().getQueryProvidersName(true)
-                : Arrays.asList(providerArr);
+        List<String> providers = PluginController.getInstance().filterDicomQueryProviders(providerArr);
+
+        if (providers.size() == 0) {
```

Comment:
I was initially thinking about only using the internal dimProvider list if the provider parameter was empty. But I suppose this works too. We'll just make sure that both Bioinformatics and BMD agree on this API.